app/server/lib/Requests.ts

```diff
@@ -80,16 +80,21 @@ export class DocRequests {
 
   private async _handleSingleRequestRaw(request: SandboxRequest): Promise<Response> {
     try {
-      if (process.env.GRIST_EXPERIMENTAL_PLUGINS != '1') {
+      if (process.env.GRIST_EXPERIMENTAL_PLUGINS != '1' || process.env.GRIST_ENABLE_REQUEST_FUNCTION != '1') {
```

Comment:
I think `GRIST_ENABLE_REQUEST_FUNCTION` should be enough on its own, without `GRIST_EXPERIMENTAL_PLUGINS`. If both are required, the README isn't reflecting that.